### DISCUSSION TITLE
is any way to work with useDeferredValue ?

### DISCUSSION THREAD

**Discussion by arvinxx** [pmndrs/zustand#1685] [Q&A]
this pattern seems not work?
```
  const value = useStore(longtimeSelector);

  const deferredValue = useDeferredValue(value);
```

**Comment by dai-shi** [comment]
It doesn't occur any errors, right? It's just that the value is never deferred, I suppose. An external store is basically "sync", and can't be deferred.

Curious how [use-zustand](https://github.com/dai-shi/use-zustand) works for your case.
